src/tls.c

```diff
@@ -7174,9 +7174,9 @@ static int TLSX_CA_Names_Parse(WOLFSSL *ssl, const byte* input,
 
 #else
 
-#define CAN_GET_SIZE(...)  0
-#define CAN_WRITE(...)     0
-#define CAN_PARSE(...)     0
+#define CAN_GET_SIZE()  0
+#define CAN_WRITE()     0
+#define CAN_PARSE()     0
```

Comment:
These should be defined to take one dummy arg, as explained in my comment below.